fineract-provider/src/main/resources/sql/migrations/core_db/V119__add_template_table.sql

```diff
@@ -31,9 +31,9 @@ CREATE TABLE `m_template` (
 DROP TABLE IF EXISTS `m_templatemappers`;
 CREATE TABLE `m_templatemappers` (
   `id` BIGINT NOT NULL AUTO_INCREMENT,
-  `mapperkey` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `mapperkey` varchar(255) COLLATE utf8mb4_general_ci DEFAULT NULL,
```

Comment:
What happens to old deployments when u update and previous migration file? Can we instead update in a newer migration script?